tests/console/docker.pm

```diff
@@ -0,0 +1,31 @@
+use base "consoletest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    become_root;
+
+    # install the docker package
+    script_run "zypper --gpg-auto-import-keys -n in docker && echo 'docker_installed' > /dev/$serialdev";
+    die "docker install failed" unless wait_serial "docker_installed", 200;
+
+    # start the docker daemon
+    script_run "systemctl start docker && echo 'docker_started' > /dev/$serialdev";
+    die "docker service start failed" unless wait_serial "docker_started", 200;
+
+    # pull the alpine image
+    script_run "docker pull alpine && echo 'docker_pull' > /dev/$serialdev";
+    die "docker pull alpine image failed" unless wait_serial "docker_pull", 300; # increase timeout, on systems using devicemapper as storage backend docker's initialization can take some time
+
```

Comment:
are you sure that's a good idea? I know there are other tests that rely on www.opensuse.org which is already error prone but this one will make the test depend on the connection to the docker hub, right?
Not that this test may be run quite often so we will hammer them.